src/openrct2/world/MapAnimation.cpp

```diff
@@ -185,6 +185,9 @@ static bool map_animation_invalidate_small_scenery(int32_t x, int32_t y, int32_t
             continue;
 
         sceneryEntry = tileElement->AsSmallScenery()->GetEntry();
+        if (sceneryEntry == nullptr)
+            continue;
```

Comment:
Alternatively you could have returned true here as there should not be another small_scenery element at this location. But then NE does like to do funny things like overlapping scenery so maybe this does make more sense.